generators/client/templates/angular/src/main/webapp/app/_vendor.ts

```diff
@@ -6,6 +6,7 @@
 // import 'angular-loading-bar';
 // import 'ng-file-upload';
 // import 'ng-infinite-scroll';
+import 'bootstrap/dist/js/bootstrap.min.js';
```

Comment:
is bootstrap using js to show hide stuff instead of css?  Yes. And I see that it needs a configuration regarding tether as an error is displayed.